content/en/docs/rules/custom-ruleset.md

```diff
@@ -0,0 +1,383 @@
+    (syscall.type = open and (fd.name contains "/etc/passwd" or fd.name contains "/etc/shadow"))
+  output: Log the relevant event.
+  priority: debug
+```
+
+In this example, the condition is structured to prioritize specific evaluations through the use of parentheses. It ensures that the rule triggers an alert when either the `execve` syscall type is matched and the process name is `/bin/bash`, or when the `open` syscall type is matched and the file descriptor name contains either `/etc/passwd` or `/etc/shadow`.
+
+The proper placement of parentheses allows for accurate evaluation and ensures that the rule behaves as intended.
+
+## False Positives and Negatives
+
+As previously explained, the `condition` key includes all the necessary checks that an event must satisfy in order to trigger a specific rule. If there are too few checks in the condition, the rule might become too general and trigger frequently, potentially resulting in a high number of false positives. These broad conditions can be useful for initial testing purposes to ensure that the rule is being reached and triggered. If the rule is never triggered, it suggests that a previous rule may be capturing the event before it reaches the intended rule.
+
+An example of a rule that is too general is provided below:
+
+```
+- rule: Too General Rule
+  desc: An example of a rule that is overly broad
+  condition: >
+    proc.name != "systemd" and evt.type = execve
+  output: Log the relevant event.
+  priority: debug
+```
+
+Another example would be a rule designed to monitor all activity generated by a specific process, like:
+
+```
+- rule: Monitor only a process named malicious
+  desc: Another example of an overly broad rule
+  condition: proc.name = malicious
+  output: The process %proc.name has used the syscall %evt.type
+  priority: debug
+```
+
+While these examples are useful for initial testing and gathering samples from certain commands, it's important to note that they are too general to yield reliable alerts. Instead, these broad examples are more likely to generate a large volume of false positives.
+
+On the contrary, if a condition becomes overly specific or contradictory, it may fail to trigger when necessary, resulting in what is known as false negatives. To illustrate this point, consider the following example:
+
+```
+- rule: Example of a too specific rule
+  ...
+  condition: evt.type = execve and proc.name = malicious and proc.pid = 1
+  output: The process %proc.name has used the syscall %evt.type
+```
+
+The previous rule would seldom trigger because it relies on a specific PID (Process ID) for the process, which may rarely occur unless executed within a container. While this example represents an extreme case, it serves to highlight the consequences of being excessively specific when defining a condition. It emphasizes that overly specific conditions can lead to infrequent triggering or potentially no triggering at all in real-world scenarios.
```

Comment:
```suggestion
The previous rule would seldom trigger because it relies on a specific PID (Process ID) for the process, which may only occur if executed within a container. While this example represents an extreme case, it highlights the consequences of being excessively specific when defining a condition. It emphasizes that overly specific conditions can lead to infrequent triggering or potentially not triggering in real-world scenarios.
```
I'm not 100% sure about this one -- my suggestion may shift the meaning slightly, but I wanted to share it anyway.